nanopb/0.3.9.1/nanopb.podspec

```diff
@@ -0,0 +1,33 @@
+Pod::Spec.new do |s|
+  s.name         = "nanopb"
+  s.version      = "0.3.901"
+  s.summary      = "Protocol buffers with small code size."
+
+  s.description  = <<-DESC
+                    Nanopb is a small code-size Protocol Buffers implementation
+                    in ansi C. It is especially suitable for use in
+                    microcontrollers, but fits any memory restricted system.
+                   DESC
+
+  s.homepage     = "https://github.com/nanopb/nanopb"
+  s.license      = { :type => 'zlib', :file => 'LICENSE.txt' }
+  s.author       = { "Petteri Aimonen" => "jpa@nanopb.mail.kapsi.fi" }
+  s.source       = { :http => "https://github.com/nanopb/nanopb/archive/#{s.version}.tar.gz" }
+  s.source       = { :git => "https://github.com/nanopb/nanopb.git", :tag => "0.3.9.1" }
+
+  s.requires_arc = false
+  s.xcconfig = { 'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) PB_FIELD_32BIT=1 PB_NO_PACKED_STRUCTS=1 PB_ENABLE_MALLOC=1' }
```

Comment:
Is changing the users xcconfig versus the pod target xcconfig (`s.pod_target_xcconfig`) intentional?